sdk/core/azure-core/azure/core/pipeline/policies/_base.py

```diff
@@ -61,11 +61,8 @@ class HTTPPolicy(abc.ABC, Generic[HTTPRequestTypeVar, HTTPResponseTypeVar]):
     :type next: ~azure.core.pipeline.policies.HTTPPolicy or ~azure.core.pipeline.transport.HttpTransport
     """
 
-    next: Union[
-        "HTTPPolicy[HTTPRequestTypeVar, HTTPResponseTypeVar]",
-        "HttpTransport[HTTPRequestTypeVar, HTTPResponseTypeVar]",
```

Comment:
This was an error introduced in my previous PR. `next` is always a policy, and can never be a transport (the transport is wrapped in a TransportRunner that implements a policy protocol)

Reply:
`next` could be a policy or a transport (TransportRunner) https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/core/azure-core/azure/core/pipeline/_base.py#L138

Reply:
Oh, I get your point. Yes. `next` could only be a policy. :)